docs/source/overview.md

```diff
@@ -0,0 +1,81 @@
+# SimPhoNy: interoperability powered by ontology
+
+SimPhoNy is an ontology-based framework aimed at enabling interoperability between different simulation and data management tools, with a focus on materials science.
+
+
+
+## What can SimPhoNy be used for?
+
+### Manipulate ontology-based linked data, a format well suited for [FAIR data principles](https://en.wikipedia.org/wiki/FAIR_data)
+
+
+[Linked data](https://en.wikipedia.org/wiki/Linked_data) is a format for structured data that facilitates the interoperability among different data sources. In particular, the data is structured as a directed graph, consistent of nodes and labeled arcs. With SimPhoNy, you can not only manipulate this linked data, **but also transform existing non-linked data into linked data**.
+
+To better understand the idea of linked data, take a quick glance at the toy example below. It shows data about a city from three different data sources: the city's traffic authority, a map from a city guide, and the university registry. As some of the concepts are present in multiple datasets, the linked data representation naturally joins all of them into a single one. 
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Sample linked data](./_static/img/mergedlinkeddata.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+_Linked data about a city from three different sources: the city's traffic authority, a map from a city guide, and the university registry. Each data source is represented using a different color and column._
+
+</figcaption>
+
+</figure>
+
+
+Moreover, in SimPhoNy, all linked data is based on [ontologies](https://en.wikipedia.org/wiki/Ontology_(information_science)), which give **meaning** to the data. Specifically, SimPhoNy works with ontologies based on the [Web Ontology Language](https://en.wikipedia.org/wiki/Web_Ontology_Language), making the data compatible with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web).
```

Comment:
> Moreover, in SimPhoNy, all linked [...]

It seems to be out of context. Meaning, moreover to what? SimPhoNy was part of the previous explanation

Reply:
That what should be "linked data". You can have just plain triples or you can have an ontology on top of that. I am not sure of the best way to convey this though.

Reply:
Same mistake of mine: "SimPhoNy was *not* part".
The comment was about the choice of the word "Moreover", which I thought doesn't quite fit there, but it's up to you.

Reply:
I think now I found the right way to express the idea. Check [f57f8ed](https://github.com/simphony/docs/pull/121/commits/f57f8edfc6300cf02a303a4e69c959db02d5649e) if interested.